src/components/Buttons/CommentsButton.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export const CommentsButton = ({ commentsLength, toggleCommentVisibility }) => {
```

Comment:
no need to write a separate component for this, you can do all this in one single button